### ISSUE TITLE
The output of the Conv2D layer in MXNet  is different from that in TensorFlow and CNTK

### ISSUE THREAD

**Issue by Justobe** [apache/incubator-mxnet#17694]
## Description

Hi, I encountered a problem when I used LeNet-5 for image classification task. The prediction result of backend MXNet is different from TensorFlow and CNTK. The input picture is sampled from the Fashion-MNIST DataSet. 

![fashion-mnist-0](https://user-images.githubusercontent.com/15032308/75327058-b740f080-58b6-11ea-87c1-084bd728c2ee.png)

The outputs ( Top-1 prediction) of three backends are shown below（Prediction label id and  Probability ）:

**TensorFlow**: 9, 1.0 

**CNTK**:  9,1.0

**MXNET**: 8,1.0

The prediction of TensorFlow and CNTK are both 'Ankle boot'(id 9) but the prediction of MXNet is 'Bag' (id 8)

It seems that there is something wrong with MXNet. Then I put image into the model and recorded the ouputs of each layers to see how this happens. It seems that the outputs of CNTK and TensorFlow are very close, but for MXNet, the outputs of a conv2d layer and all the layers after this layer are very different from CNTK and TensorFlow.

The results are attached below.

TensorFlow vs CNTK

![tf-cntk](https://user-images.githubusercontent.com/15032308/75327087-c162ef00-58b6-11ea-9e89-cab69945dd60.png)

MXNet vs TensorFlow 

![tf-mxnet](https://user-images.githubusercontent.com/15032308/75327100-c627a300-58b6-11ea-94cf-16f902153f72.png)

MXNet vs CNTK

![cntk-mxnet](https://user-images.githubusercontent.com/15032308/75327108-ca53c080-58b6-11ea-8158-20509448f1cc.png)

The delta column shows the discrepancy, which is calculated as:
![MAD](https://user-images.githubusercontent.com/15032308/75327123-cfb10b00-58b6-11ea-9d83-1a47bfc6ee99.png)

Based on the results, the deviation seems to be caused by conv2d of MXNet backend.

## To Reproduce
The code of getting prediction result of each backend  is really simple. 

get_prediction.py :

```python
import os
import sys
import numpy as np
from PIL import Image
bk = sys.argv[1]
os.environ['KERAS_BACKEND'] = bk
os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
os.environ["CUDA_VISIBLE_DEVICES"] = "0"

import keras
from keras import backend as K
print("Using backend :{}".format(K.backend()))

base_model = keras.models.load_model("lenet5-fashion-mnist_origin.h5")
adam = keras.optimizers.Adagrad(lr=0.01, epsilon=None, decay=0.0)
base_model.compile(loss='categorical_crossentropy', optimizer=adam, metrics=['accuracy'])

imagename = 'fashion-mnist-0.png'
img = Image.open(imagename)
x_test = np.array(img)
x_test = x_test.reshape(28,28,1)
select_data = np.expand_dims(x_test, axis=0)

prediction = base_model.predict(select_data)
pred = np.argsort(prediction[0])
print(pred)
print(f"Prediction of {bk}: {pred[-1]},{prediction[0][pred[-1]]}")

```

You can run the script like this ():

> python get_prediction.py tensorflow

Change tensorflow to 'mxnet' and 'cntk'

I provide scripts to get outputs of the middle layers and calculate the difference:

[code.zip](https://github.com/apache/incubator-mxnet/files/4254364/code.zip)

Thanks in advance.

## What have you tried to solve it?

I encountered this problem on mxnet-cu101(Version 1.5.1.post0). When I upgraded mxnet to 1.6, the bug still exists.

## Environment

- MXNet: mxnet-cu101==1.5.1.post0
- keras-mxnet: 2.2.4.2
- CUDA: 10.1

You can use the following command to configure the environment

```
pip install keras-mxnet
pip install mxnet-cu101
pip install tensorflow-gpu==1.14.0
pip install cntk-gpu==2.7
```
